MiniSim/Extensions/UserDefaults+Configuration.swift

```diff
@@ -46,4 +47,9 @@ extension UserDefaults {
         get { bool(forKey: Keys.enableAndroidEmulators) }
         set { set(newValue, forKey: Keys.enableAndroidEmulators) }
     }
+// Need to add ui for changing the terminal preference
+    @objc public dynamic var preferedTerminal: String? {
+        get { string(forKey: Keys.preferedTerminal) }
```

Comment:
Also can you add test for this in https://github.com/okwasniewski/MiniSim/blob/main/MiniSimTests/UserDefaultsTests.swift ?